packages/toml/src/index.ts

```diff
@@ -0,0 +1,66 @@
+import { Taplo } from '@taplo/lib'
+import type { Plugin } from 'prettier'
+
+import { defaultOptions, options } from './options'
```

Comment:
`options` is shadowed by several functions below, can we rename it? For example `prettierOptionsDefinitions`?